### ISSUE TITLE
[crash] glyphView keyboard input

### ISSUE THREAD

**Issue by davelab6** [trufont/trufont#56]
When I run the latest git master, and press a key to jump to a encoded glyph in the glyphView, I get a crash:
```py
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/defconQt/glyphCollectionView.py", line 274, in keyPressEvent
    unicodedata.category(event.text()) != "Cc"):
TypeError: category() argument must be a unicode character, not str
Abort trap: 6
```

**Comment by adrientetar** [comment]
You are not running the latest git master since this was fixed in 8d2f535a8b355c41abed30d62a2647ae0de2df8a.

**Comment by davelab6** [comment]
Ah, I was running `python3 -m defconQt` in the wrong place. Sorry!
